src/lib/database/bot.ts

```diff
@@ -0,0 +1,72 @@
+import { container } from '#root/Container';
+import { logger } from '#lib/structures';
+import { Bot } from '@prisma/client';
+const { db } = container;
+
+/**
+ * Create a bot entry to the database
+ * @param id The bot Id
+ * @param data The data to create the bot with
+ */
+export async function createBot(id: string, data?: Partial<Omit<Bot, 'id'>>) {
+	try {
+		return await db.bot.create({
+			data: {
+				id: id,
+				...data,
+			},
+		});
+	} catch (err) {
+		const error = err as Error;
+		logger.error(error, error.message);
+	}
```

Comment:
this is a nice try catch block pattern that could be abstracted out for reuse. similar to the `.unwrap_or_log()` you see in rust with logging.